### PR TITLE
Update version on master

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
 	<repository type="git">https://github.com/nextcloud/gallery.git</repository>
 	<dependencies>
 		<php min-version="5.6" max-version="7"/>
```
Comment (nickvergessen) [inline appinfo/info.xml]: @oparoz does this work with php 7.1 and upcoming 7.2?

Comment (nickvergessen) [reply appinfo/info.xml]: It should support the same versions as nextcloud 14 because its shipped

Comment (oparoz) [reply appinfo/info.xml]: No problem to make it compatible with 7.2 for 14

Comment (oparoz) [reply appinfo/info.xml]: 14 is 7+, no?`If that's the case, let's update min-max to 7-7.2

Comment (rullzer) [reply appinfo/info.xml]: 14 is 7.0 - 7.2 for now yes

**Pull request merged**